backend/src/comment/comment.controller.ts

```diff
@@ -35,4 +35,9 @@ export class CommentController implements CrudController<Comment> {
       repositoryId: commentEvent.repository.id,
     });
   }
+
+  @Get('projects')
```

Comment:
I'm not sure about this, I think in the end we will have a table named "projects" and link each project to users depending on which project they can access. Or store it in a column of the user table. This route should not be here. you can imagine a route like @Get('commentsFilterByProjects')  but not something parsing the comments db and returning the projects ids